bindings/ios/MEGABackgroundMediaUpload.h

```diff
@@ -91,6 +91,20 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (NSString *)uploadURLString;
 
+/**
+ * @brief Sets the GPS coordinates for the node
+ *
+ * The node created via completeBackgroundMediaUpload:fileName:parentNode:fingerprint:originalFingerprint:binaryUploadToken:delegate: in MEGASdk
+ * will gain these coordinates as part of the
+ * node creation. If the unshareable flag is set, the coodinates are encrypted in a way that even if the
+ * node is later shared, the GPS coordinates cannot be decrypted by a different account.
+ *
+ * @param latitude The GPS latitude
+ * @param latitude The GPS longitude
+ * @param unshareable Set this true to prevent the coordinates being readable by other accounts.
+ */
+- (void)setCoordinatesWithLatitude:(NSNumber *)latitude longitude:(NSNumber *)longitude isUnshareable:(BOOL)unshareable;
```

Comment:
Maybe it is better use double directly as parameters. What do you think?
(I know there is a lot of places where we use NSNumbers instead primitive types)

Reply:
This way, in the app layer code, we don't need create a `NSNumber` from a double, that finally will be convert to a double when pass it to the SDK

Reply:
In `MEGASdk` we need to use `NSNumber` to add ability to remove coordinates from nodes as described in MegaApi.
>To remove the existing coordinates, set both the latitude and longitude to the value MegaNode::INVALID_COORDINATE.

I think it would be good to keep consistent? Especially if you want to remove coordinates later on.

Reply:
You meant keep consistency between: `setNodeCoordinates` and `setCoordinatesWithLatitude` ?

I would use `double` in both cases as parameters.
If we would want remove the coordinates we can use the `MegaNode::INVALID_COORDINATE` as parameter.

Reply:
Yes, I was to keep consistency with `setNodeCoordinates` and `setCoordinatesWithLatitude`. But here is a different case where we don't even have a node. So, agree with you. I have changed to `double`.